src/Employer/Employer.Web/Views/Part1/Location/Location.cshtml

```diff
@@ -1,68 +1,91 @@
 @using Esfa.Recruit.Employer.Web.ViewModels.Part1.Location;
-@using Esfa.Recruit.Shared.Web.Mappers;
+@using Esfa.Recruit.Shared.Web.Configuration
+@using Microsoft.Extensions.Options
+@inject IOptions<PostcodeAnywhereConfiguration> PcaConfig
+
 @model LocationViewModel;
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <a asp-show="@Model.CanShowBackLink" asp-route="@RouteNames.EmployerName_Get" esfa-automation="link-back" class="govuk-back-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">Back</a>
-        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState }'/>
+        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState }' />
```

Comment:
Also need to add `, OrderedFieldNames = Model.OrderedFieldNames` so Postcode is listed at the bottom of the errors. (At the moment its shown as the first error).

Reply:
fixed